website/docs/intro.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 1
+title: Introduction
+---
+
+### What is odo?
+
+odo is a fast, iterative and straightforward CLI tool for developers who write, build, and deploy applications on Kubernetes.
+
+odo abstracts the complex Kubernetes terminology so that an application developer can focus on writing code in their favourite framework without having to learn Kubernetes.
+
+odo is focused on [inner loop](./intro#what-is-inner-loop-and-outer-loop) development with some tooling that would help users transition to the [outer loop](./intro#what-is-inner-loop-and-outer-loop).
+
+Brendan Burns, the cofounder of Kubernetes, said in the [book Kubernetes Patterns](https://www.redhat.com/cms/managed-files/cm-oreilly-kubernetes-patterns-ebook-f19824-201910-en.pdf):
+
+> It (Kubernetes) is the foundation on which applications will be built, and it provides a large library of APIs and tools for building these applications, but it does little to provide the application architect or developer with any hints or guidance for how these various pieces can be combined into a complete, reliable system that satisfies their business needs and goals.
+
+odo makes Kubernetes easy for application architects and developers.
+
+### What is "inner loop" and "outer loop"?
+
+The inner loop consists of local coding, building, running, and testing the application—all activities that you, as a developer, can control. The outer loop consists of the larger team processes that your code flows through on its way to the cluster: code reviews, integration tests, security and compliance, and so on. The inner loop could happen mostly on your laptop. The outer loop happens on shared servers and runs in containers, and is often automated with continuous integration/continuous delivery (CI/CD) pipelines. Usually, a code commit to source control is the transition point between the inner and outer loops.
+*([Source](https://developers.redhat.com/blog/2020/06/16/enterprise-kubernetes-development-with-odo-the-cli-tool-for-developers#improving_the_developer_workflow))*
+### Who should use odo?
+
+You should use odo if:
+* you are developing applications using Node.js, Spring Boot, or similar framework
```

Comment:
Can we add a link to the full list of frameworks supported by odo?

Reply:
I'm not aware if there's a link to it. But there could be a separate doc that lists the supported frameworks. Note that when you call something as "supported", it means something different from what's conveyed in this line.